index.md

```diff
@@ -56,7 +56,6 @@ Contains information about the 2i2c team and our projects, and some useful resou
 :maxdepth: 1
 :caption: Team Reference
 reference/projects
-reference/hubs
```

Comment:
Imagined that we would leave this page, but just point it to the pilot hubs repo. That way the source of truth is still the team compass, it just delegates to pilot hubs. Maybe we can just add a section to the "team coordination" page?

Reply:
> Maybe we can just add a section to the "team coordination" page?

Can you expand on that a little?

Reply:
I mean that if we think it is a bit much to have a dedicated page that simply links to the hubs table in this repo, we could turn that page into a short section here https://team-compass.2i2c.org/en/latest/practices/coordination.html

Reply:
@choldgraf that makes sense - let's do that!